LANGUAGES.md

```diff
@@ -0,0 +1,24 @@
+## Languages
```

Comment:
Format the file extensions to make it **Bold** and fit it within a table 

**E.g**

|Extension | Name |
|-|-|
|**js**|javascript|